examples/jsm/loaders/KTX2Loader.js

```diff
@@ -555,6 +563,46 @@ class KTX2Container {
 
 	}
 
+	getTexFormat () {
+
+		var TextureFormat = this.basisModule.TextureFormat;
+
+		return this.dfd.colorModel === DFDModel.UASTC ? TextureFormat.UASTC4x4 : TextureFormat.ETC1S;
+
+	}
+
+	getAlpha () {
+
+		var TextureFormat = this.basisModule.TextureFormat;
+
+		// TODO(donmccurdy): Handle all channelIDs (i.e. the R & R+G cases),
+		// choosing appropriate transcode target formats or providing queries
+		// for applications so they know what to do with the content.
+
+		if ( this.getTexFormat() === TextureFormat.UASTC4x4 ) {
+
+			// UASTC
+
+			if ( ( this.dfd.samples[ 0 ].channelID & 0xF ) === DFDChannel.UASTC.RGBA ) {
+
+				return true;
+
+			}
+
+			return false;
+
+		}
+
+		// ETC1S
+
+		if ( this.dfd.numSamples === 2 && ( this.dfd.samples[ 1 ].channelID & 0xF ) === DFDChannel.ETC1S.AAA ) {
+
+			return true;
+
+		}
+
```

Comment:
Missing `return false;` for the non-alpha ETC1S case.

Reply:
Fixed, thanks!